portfolio/src/main/webapp/style.css

```diff
@@ -115,6 +115,16 @@ main > section > .entry > .description > * {
   flex: 1 0 0;
 }
 
+.comments-container > .comments-form-container.logged-in > .login,
+.comments-container > .comments-form-container > .comments-form {
```

Comment:
do we need this many selectors? can we reduce them here and below?

Reply:
Reduced in latest commit